.readthedocs.yaml

```diff
@@ -0,0 +1,34 @@
+# .readthedocs.yaml
+# Read the Docs configuration file
+# See https://docs.readthedocs.io/en/stable/config-file/v2.html for details
+
+# Required
+version: 2
+
+# Set the version of Python and other tools you might need
+build:
+  os: ubuntu-22.04
+  tools:
+    python: "3.8"
+    # You can also specify other tool versions:
+    # nodejs: "19"
+    # rust: "1.64"
+    # golang: "1.19"
+
+# Build documentation in the docs/ directory with Sphinx
+sphinx:
+  configuration: docs/conf.py
+# fail_on_warning: true
```

Comment:
Can we make a new story to re-enable fail on warnings or associate with the ticket if we have one to fix the warningss already

Reply:
@kebringer-cb does a ticket already exist for fixing all the warnings in that build?
(Kylie told me that some of those doc fixes are not easy, she's looked at them.)

Reply:
There is now: CBAPI-4705